main.py

```diff
@@ -135,4 +138,4 @@ def parse_arguments():
         if os.getenv("FLASK_DEBUG") == "True":
             debug_val = True
 
-        app.run(port=5000, debug=debug_val, extra_files=extra_files)

```

Comment:
Apparently AirPlay uses port 5000, and makes the server fail.
https://takamoto.biz/2022/02/%E3%80%90mac-monterey%E3%80%91flask%E3%81%AE%E3%83%87%E3%83%95%E3%82%A9%E3%83%AB%E3%83%88%E3%83%9D%E3%83%BC%E3%83%885000%E3%81%A7%E6%8E%A5%E7%B6%9A%E3%82%A8%E3%83%A9%E3%83%BC/